docs/preview/features/secret-store/index.md

```diff
@@ -12,6 +12,18 @@ Once register, you can fetch all secrets by using `ISecretProvider` which will g
 
 > :bulb: See [this section](#using-secret-store-within-azure-functions) if you want to use the secret store functionality whitin Azure Functions.
 
+## Why would I use it?
+Why would you use our Arcus secret store instead of just using the Azure SDK directly to access Azure Key Vault secrets?
+The Arcus secret store has some advantages over using the Azure SDK or `IConfiguration` directly:
```

Comment:
I would add "plug & play" so that you can mix and match what you want?

Reply:
Hmm, isn't somewhat the same as extensibility? I changed the multiple locations somewhat to include this.